src/helpers.cpp

```diff
@@ -197,9 +197,132 @@ ada_really_inline uint64_t swap_bytes_if_big_endian(uint64_t val) noexcept {
 #endif
 }
 
+ada_really_inline int trailing_zeroes(uint32_t input_num) {
```

Comment:
Can you add a comment on top of this function? If we don't want to expose this, we should add private to comments.

Reply:
I did. In my view, an inline function defined in a source file and just used locally does not have to be declared in the header file (and indeed, doing so just adds noise, isn't it?).

Reply:
(To be clearer: I did add a comment in a later commit.)